__tests__/logMeasures.js

```diff
@@ -49,13 +49,13 @@ function logToConsole({ port, components }, measures) {
   if (!components) {
     logMeasures(measures)
   } else if (typeof components !== undefined && Array.isArray(components)) {
-    var stats = getRequiredMeasures(components, measures)
+    var stats = getMeasuresByComponentName(components, measures)
 
     logMeasures(stats)
   }
 }
 
-function getRequiredMeasures(components, measures) {
+function getMeasuresByComponentName(components, measures) {
```

Comment:
I noticed with this change that you actually had duplicate implemetation on the test (?)
why is that?

Reply:
In `src/npm/hook.js` we are send them to a server listening at a particular port while in tests, we are treating them as a proxy i.e logging it to the console. Anyway, it does makes sense to remove this redundant implementation but I think its fine. Let me know what you think!

Reply:
I would think that the less redundant code, the better to reduce duplicate work and increase coverage. Let's take a look at this another time. Leaving this as is doesn't break anything